Makefile

```diff
@@ -178,7 +178,7 @@ rpc-format:
 	@$(call print, "Formatting protos.")
 	cd ./lnrpc; find . -name "*.proto" | xargs clang-format --style=file -i
 
-rpc-check: rpc-format rpc
+rpc-check: rpc
```

Comment:
Remove this thing completely?

Reply:
Or is the check just building the rpcs and seeing if that succeeds?

Reply:
No, we still want to check that the protos are _compiled_ with the correct version. There we have a strict requirement and instructions for installing the right version of the tools/libs.